coding/python_testing.rst

```diff
@@ -110,14 +110,18 @@ LSST Utility Test Support Classes
 Special Asserts
 ---------------
 
-Inheriting from :lclass:`lsst.utils.tests.TestCase` rather than :class:`unittest.TestCase` enables two new asserts that are useful for doing element-wise comparison of two :mod:`numpy` arrays.
+Inheriting from :lclass:`lsst.utils.tests.TestCase` rather than :class:`unittest.TestCase` enables two new asserts that are useful for doing element-wise comparison of two :mod:`numpy`-like arrays.
 
-:lmeth:`~lsst.utils.tests.TestCase.assertClose`
-   Asserts that the corresponding elements in two arrays are within the specified tolerance.
-:lmeth:`~lsst.utils.tests.TestCase.assertNotClose`
-   Asserts that the two arrays are different.
+`lsst.utils.tests.TestCase.assertFloatsAlmostEqual <https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/namespacelsst_1_1utils_1_1tests.html#a09ee2482a2e8d71e8612c0378f4286fc>`_
+   Asserts that floating point scalars and/or arrays are equal are within the specified tolerance. The default tolerance is significantly tighter than the tolerance used by `unittest.TestCase.assertAlmostEqual` or `numpy.testing.assert_almost_equal`; if you are replacing either of those methods you may have to specify `rtol` and/or `atol` to prevent failing asserts.
```

Comment:
You'll need to add a `:meth:` in from of the `assertAlmostEqual` and `assert_almost_equal` expressions in unittest and numpy.